cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -114,6 +121,21 @@ func trackChunks(testData []byte, submitMetrics bool) error {
 				return
 			}
 
+			var hive string
+			err = rpcClient.Call(&hive, "bzz_hive")
+			if err != nil {
+				log.Error("error calling rpc client", "err", err, "host", httpHost)
+				hasErr = true
+				return
+			}
+
+			bzzAddr := strings.Split(strings.Split(hive, "\n")[3], " ")[10]
```

Comment:
This is rather ugly - do we have an API that returns the bzz addr for a given host? `admin.NodeInfo` doesn't have that information, it has only `id` and `enode`.